.travis.yml

```diff
@@ -12,9 +12,9 @@ install:
   - npm install -g polymer-cli
```

Comment:
You've included deps in a package.json, do you also need to run `npm i` here?

Reply:
Already been added